### PR TITLE
adding explicit parameter types to string parameters

### PR DESCRIPTION
**Pull request by jbohren**

### CODE REVIEW

File: src/gscam.cpp [hunk 1]
```diff
@@ -68,7 +68,7 @@ namespace gscam {
     }
 
     // Get additional gscam configuration
-    declare_parameter("sync_sink", true);
+    declare_parameter("sync_sink",  true);
```
Comment (wep21) [inline src/gscam.cpp:71]: @jbohren I don't think this space is necessary.

Comment (jbohren) [reply src/gscam.cpp:71]: Neither do I! You're welcome to fix it.

**Comment by clydemcqueen** [comment]
This doesn't build on Foxy. Looks like @wep21 found a clean solution that builds on Foxy, Galactic and Rolling, so that is preferred.

**Comment by jbohren** [comment]
> This doesn't build on Foxy. Looks like @wep21 found a clean solution that builds on Foxy, Galactic and Rolling, so that is preferred.

Sounds good, thanks for the follow-up.

**Pull request closed without merging**